### PR TITLE
Export color factory function

### PR DESCRIPTION
**Pull request by martenbjork**
### WHY are these changes introduced?
Other programs should be able to use the color factory.

### WHAT is this pull request doing?
It adds the `buildColors` to the list of exports.

### CODE REVIEW

**Comment by martenbjork** [comment]
Smallest PR ever. For now, this allows an external program to import the color factory and consume the CSS Custom Properties.

The color factory and the tests are currently built with in CSS Custom Properties in mind. In the future, we may want to separate the factory function from the CSS transformations. That way, you could run the factory and then, separately transform the output to an object, CSS Properties or Tokens — whichever suits your use case.

**Comment by kaelig** [comment]
If this is something we'll host as part of polaris-tokens later on, would it make sense to make it clear this is an "unstable" feature?

**Comment by martenbjork** [comment]
> If this is something we'll host as part of polaris-tokens later on, would it make sense to make it clear this is an "unstable" feature?

Absolutely! Updated. (Using the right conventions?)

File: src/index.ts [hunk 1]
```diff
@@ -11,4 +11,6 @@ export {
 export {
   WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT,
 } from './utilities/within-content-context';
+
+export {buildColors as unstableBuildColors} from './utilities/theme/utils';
```
Comment (tmlayton) [inline src/index.ts]: let's make it _really_ clear :) 

```suggestion
export {buildColors as UNSTABLE_BuildColors} from './utilities/theme/utils';
```

File: src/index.ts [hunk 2]
```diff
@@ -11,4 +11,6 @@ export {
 export {
   WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT,
 } from './utilities/within-content-context';
+
+export {buildColors as UNSTABLE_BuildColors} from './utilities/theme/utils';
```
Comment (tmlayton) [inline src/index.ts]:
```suggestion
export {buildColors as UNSTABLE_BuildColors} from './utilities/theme';
```

and add an export from `theme/index.ts`, this allows us to quickly look at the local index file to have an idea of what we expose publicly

**Pull request merged**